docs/lib/analytics/fragments/flutter/getting-started/10_preReq.md

```diff
@@ -6,6 +6,6 @@
 
 <amplify-callout warning>
 
-If you are using Flutter (2.2.0), you need to disable sound null safety by following the instructions [here](https://dart.dev/null-safety/unsound-null-safety#testing-or-running-mixed-version-programs). Null safety support for Amplify Flutter is being worked on actively.
+If you are using Flutter (2.2.0 or greater), you need to disable sound null safety by following the instructions [here](https://dart.dev/null-safety/unsound-null-safety#testing-or-running-mixed-version-programs). Null safety support for Amplify Flutter is being worked on actively.
```

Comment:
Nit suggestion:
```suggestion
If you are using Flutter (2.2.0 or greater), you need to disable sound null safety by following the instructions [here](https://dart.dev/null-safety/unsound-null-safety#testing-or-running-mixed-version-programs). Null safety support for Amplify Flutter is actively being worked on.
```
Feel free to ignore as it is debatable if this really flows better and whether we wish to put emphasis on the adverb or the action